.github/workflows/tools-tests.yml

```diff
@@ -1013,3 +1013,27 @@ jobs:
           sed -n 's/^<.*>\s\+\S\+\s\+\(\S\+\).*/\1/p' output | sort > actual
           certutil -K -d /root/.dogtag/nssdb | sed -n 's/^<.*>\s\+\S\+\s\+\(\S\+\).*/\1/p' | sort > expected
           diff actual expected
+
+  rpminspect-test:
+    name: Run RPMInspect
+    needs: [init, build]
+    runs-on: ubuntu-latest
+    container: registry.fedoraproject.org/fedora:${{ matrix.os }}
+    strategy:
+      matrix: ${{ fromJSON(needs.init.outputs.matrix) }}
+    steps:
+      - name: Download PKI packages
+        uses: actions/download-artifact@v2
+        with:
+          name: pki-build-${{ matrix.os }}
+          path: build/RPMS
+
+      - name: Install RPMInspect
+        run: |
+          dnf install -y dnf-plugins-core
+          dnf copr enable -y copr.fedorainfracloud.org/dcantrell/rpminspect
+          dnf install -y rpminspect rpminspect-data-fedora
+
+      - name: Run RPMInspect
+        run: |
+          rpminspect-fedora -v build/RPMS/dogtag-pki-11*.rpm
```

Comment:
Is it possible to remove the version number from this line so it will work with all versions?

Reply:
I knew you were going to ask this :laughing: I could probably regex it.

Reply:
If I do `rpminspect-fedora -v build/RPMS/dogtag-pki-1*.rpm` it still works. That gets you from `F28` to some version about 20 years from now - good enough?

Reply:
I did it properly in the end: `rpminspect-fedora -v build/RPMS/dogtag-pki-[0-9]*.rpm`